Source/WebKit/UIProcess/ios/UIKitUtilities.mm

```diff
@@ -228,6 +228,15 @@ - (UIScrollView *)_wk_parentScrollView
     return nil;
 }
 
+- (BOOL)_wk_isParentOf:(UIView *)view
```

Comment:
nit: This reads more like `isAncestorOf:`, but maybe `isParentOf:` is more idiomatic

Reply:
Oh, I can rename this to `-isAncestorOf:`. It's true that `-isParentOf:` doesn't immediately convey this.